### ISSUE TITLE
Add a debug option to pays-run

### ISSUE THREAD

**Issue by aidanheerdegen** [payu-org/payu#340]
@harshula wanted to run a model directly, without `payu` running it.

I provided the following instructions:

1. `payu run` and copy the PBS submission `qsub line` and cancel (`^c`) or delete the job (`qdel`)
2. paste the PBS submission command (without the `payu-run` command) and append `-I` to run the command interactively and wait for the job to start
3. once it has started run paste in the `payu-run` command from the PBS submit, e.g.
```g/data/hh5/public/apps/miniconda3/envs/analysis3-23.01/bin/python3.9 /g/data3/hh5/public/apps/miniconda3/envs/analysis3-23.01/bin/payu-run```
4. Wait until the `mpirun` command is printed to `STDOUT` cancel the job (`^c`) and copy and paste the actual `mpirun` command, e.g.
```
mpirun --mca io ompio --mca io_ompio_num_aggregators 1 -wdir /scratch/tm70/aph502/access-om2/work/1deg_jra55_iaf/atmosphere -np 1  /scratch/tm70/aph502/access-om2/work/1deg_jra55_iaf/atmosphere/yatm_5b6c697d.exe : -wdir /scratch/tm70/aph502/access-om2/work/1deg_jra55_iaf/
ocean -np 216  /scratch/tm70/aph502/access-om2/work/1deg_jra55_iaf/ocean/fms_ACCESS-OM_af3a94d4_libaccessom2_5b6c697d.x : -wdir /scratch/tm70/aph502/access-om2/work/1deg_jra55_iaf/ice -np 24  /scratch/tm70/aph502/access-om2/work/1deg_jra55_iaf/ice/cice_auscom_360x300_24p_2572851d_libaccessom2_5b6c697d.exe
```

These steps could be easily automated, which would be a useful service and make it really easy to run the configuration directly, for debugging purposes.

**Comment by harshula** [comment]
An option that generates a PBS script would be very useful. i.e. `$ qsub -q <queue> -P <project> <pbs-script-generated-by-payu>`

**Comment by aidanheerdegen** [comment]
> An option that generates a PBS script would be very useful

Yes it would, but `payu` doesn't work that way. For example, `payu run` submits a PBS job that calls `payu-run` within it, and passes state information to the `payu-run` call through environment variables, so things like the current run number.

My thought was to just include a `-I` flag to make it interactive as a first basic step.

**Comment by harshula** [comment]
Is it technically possible for Payu to generate a PBS script?

**Comment by aidanheerdegen** [comment]
I guess it *could* but the environment information is passed in command line arguments, so it would require writing all that stuff to a script. I don't think this gives a much better outcome than an interactive session with the working directory populated and the relevant command printed to stdout, so you can copy and paste and run it directly.

**Comment by aidanheerdegen** [comment]
Wanted to bump this because @manodeep was also wanting some similar functionality.

> These steps could be easily automated, which would be a useful service and make it really easy to run the configuration directly, for debugging purposes.

I was thinking about it, and at its most basic it is just adding `-I` into the `qsub` call. However this would run the model too ... so even better would be to not run the model command, but echo it to the screen with something like "Copy and paste this command to run the model ...".

If the user wanted to run `payu setup` themselves and then run the actual `mpirun` command it would be a little more difficult .. but they can invoke the `pay-run` command, see the `mpirun` command echo'ed to STDOUT, kill the job, `sweep` and then `setup` and run it manually.
